From d8ac6c6250985a5e1d549e358d86b737826f6686 Mon Sep 17 00:00:00 2001
From: Oleg Karfich <oleg.karfich@wago.com>
Date: Fri, 12 Oct 2018 11:57:45 +0200
Subject: [PATCH] arm: configs: am35xx_pfc_mlo: deactivate meminfo to reduce
 image size

The am35xx based devices have only about 64kB SRAM size. If the image exceeds the
size above ~61950 bytes the mlo does not boot anymore. Remove meminfo print on
boot to minimize the image size to ~61633 bytes.

Signed-off-by: Oleg Karfich <oleg.karfich@wago.com>
---
 arch/arm/configs/am35xx_pfc_mlo_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/am35xx_pfc_mlo_defconfig b/arch/arm/configs/am35xx_pfc_mlo_defconfig
index 7a0ff21..beadb21 100644
--- a/arch/arm/configs/am35xx_pfc_mlo_defconfig
+++ b/arch/arm/configs/am35xx_pfc_mlo_defconfig
@@ -3,6 +3,7 @@ CONFIG_OMAP_BUILD_IFT=y
 CONFIG_MACH_WAGO_PFC_AM35XX=y
 CONFIG_THUMB2_BAREBOX=y
 # CONFIG_ARM_EXCEPTIONS is not set
+# CONFIG_MEMINFO is not set
 CONFIG_MMU=y
 CONFIG_STACK_SIZE=0xc00
 CONFIG_MALLOC_SIZE=0x0
-- 
2.7.4

